light_curves/code/TESS_Kepler_functions.py

```diff
@@ -26,19 +26,19 @@ def TESS_Kepler_get_lightcurves(coords_list, labels_list, radius):
     
     df_lc = MultiIndexDFObject()
     #for all objects
-    for ccount, coord in enumerate(tqdm(coords_list)):
+    for objectid, coord in tqdm(coords_list):
```

Comment:
It's rather unfortunate that one has to loop through one by one, I would open a feature request to lightkurve to accept vector queries.

Reply:
moved to issue #104